dotCMS/src/main/java/com/dotmarketing/util/ImportUtil.java

```diff
@@ -1476,7 +1475,7 @@ private static void importLine ( String[] line, String currentHostId, Structure
                 }
             }
         } catch (Exception e) {
-            Logger.error(ImportUtil.class,e.getMessage(),e);
+            Logger.warn(ImportUtil.class,e.getMessage());
```

Comment:
If we don't want this error in the logs I would remove it instead of change it to warn, is not a warning, is an error.
But to be honest I don't see the problem on having the extra error log in there.

Reply:
I thought the same, more logs better but was a Bryan and Will call